Makefile

```diff
@@ -9,7 +9,7 @@ export IMAGE_VERSION = v1.39.2
 export SIMPLE_VERSION = $(shell (test "$(shell git describe --tags)" = "$(shell git describe --tags --abbrev=0)" && echo $(shell git describe --tags)) || echo $(shell git describe --tags --abbrev=0)+git)
 export GIT_VERSION = $(shell git describe --dirty --tags --always)
 export GIT_COMMIT = $(shell git rev-parse HEAD)
-export K8S_VERSION = 1.31.0
+export K8S_VERSION = 1.32.1
```

Comment:
I don't think this is correct, since CI is failing.

Reply:
It should not be added sorry

Reply:
Fixed 👍 That is because I fix it when I was working on the PR to bump kb.
Thank you . Good cacther 🥇